src/web/components/Core/StatusPopup.scss

```diff
@@ -2,64 +2,50 @@
   position: fixed;
   top: 23px;
   right: 31px;
-  padding: 24px;
-  border-radius: 5px;
+  width: 345px;
+  padding: 20px;
+  border-radius: 10px;
   z-index: 1000;
   background: var(--theme-background-content);
   box-shadow: 0px 6px 12px var(--theme-box-shadow);
-  border-radius: 4px;
-  border-left: 8px solid;
-
-  &.Success {
-    border-color: var(--theme-success);
-  }
-
-  &.Error {
-    border-color: var(--theme-error);
-  }
-
-  &.Info {
-    border-color: var(--theme-action);
-  }
-}
-
-.popup-close-button {
-  position: absolute;
-  top: 0px;
-  right: 0px;
-  background: none;
-  border: none;
-}
-
-.popup-close-icon {
-  width: 7.5px;
+  border-radius: 10px;
 }
 
 .status-popup-title-container {
   margin-bottom: 4px;
-  &.Success {
-    color: var(--theme-success);
-  }
+  display: flex;
+  justify-content: space-between;
```

Comment:
nit: I think we could also do with a 
```
align-items: center;
```
to ensure the close button lines up with the title

Reply:
This doesn't alight it the same way:  
![image](https://github.com/IABTechLab/uid2-self-serve-portal/assets/37344964/bf8da757-82cd-42d1-a15a-20c1ce520fd8)

Reply:
hmm that's weird, mine looks like this

![image](https://github.com/IABTechLab/uid2-self-serve-portal/assets/137864392/070c1bac-3c78-452c-88dd-bcc9c27abcd8)

Reply:
My bad.  I replaced one rule with the other rather than having them both.  I changed this.